### PR TITLE
Fix #3 new windows have incorrectly sized fonts

### LINKED ISSUE #3: Creating a new window (make-frame-command) doesn't use the current text scale
With scaled text run `make-frame-command`, this uses the default font size.

If this is intended behavior, keeping the current font size would be useful as a preference.

### PR DESCRIPTION
**Pull request by ideasman42**
This fixes #3.

### CODE REVIEW

File: default-text-scale.el [hunk 1]
```diff
@@ -49,6 +49,13 @@
 (defvar default-text-scale--complement 0
   "Stores the delta needed to get back to the original default face height.")
 
+(defun default-text-scale--update-for-new-frame (f)
+  "Update for new window, needed for new windows to have the correct font size
+See: https://github.com/purcell/default-text-scale/issues/3."
```
Comment (purcell) [inline default-text-scale.el]: Please don't include links to specific issues in the docstring. The commit message is sufficient for this.

File: default-text-scale.el [hunk 2]
```diff
@@ -49,6 +49,13 @@
...
+  (when (display-multi-font-p f)
+    (face-spec-recalc 'default f))
+  )
```
Comment (purcell) [inline default-text-scale.el]: This paren is lonely - please could you move it up to live with its friends?

File: default-text-scale.el [hunk 3]
```diff
@@ -125,8 +132,12 @@ default to which subsequent sizes would be reset."
             (define-key map (kbd "C-M-0") 'default-text-scale-reset)
             map)
   (if default-text-scale-mode
-      (setq default-text-scale--complement 0)
-    (default-text-scale-reset)))
+      (progn
+        (add-hook 'after-make-frame-functions #'default-text-scale--update-for-new-frame)
+        (setq default-text-scale--complement 0))
+    (progn
```
Comment (purcell) [inline default-text-scale.el]: This `progn` wrapper is unnecessary.

**Review by purcell — CHANGES_REQUESTED** [review]
Thanks - looks good, but I noted a couple of mostly cosmetic issues.

**Comment by ideasman42** [comment]
Committed changes that should resolve all issues raised (please squash this commit before merging).

**Comment by purcell** [comment]
Thanks, edited a little and merged! :-)

**Pull request merged**